test/integration/test_integration_basics.py

```diff
@@ -69,6 +68,46 @@ def test_http_h1(http_test_server_fixture):
   asserts.assertEqual(len(counters), 12)
 
 
+@pytest.mark.parametrize(
+    'server_config',
+    ["nighthawk/test/integration/configurations/nighthawk_http_origin_v2_api.yaml"])
+def test_nighthawk_test_server_v2_api(http_test_server_fixture_v2_api):
```

Comment:
Comment about including "envoy" and "deprecated" applies here and below too.

Consider renaming the config file also.